app.js

```diff
@@ -0,0 +1,157 @@
+// 基礎編
+// Q1
+let nickname = 'ごっしー';
+let age = '28';
+let greet = '私のニックネームは' + nickname + 'です。年齢は' + age + '歳です。';
+console.log(greet);
```

Comment:
基礎編Q1
よく書けてますね！👍

[FYI]
ちなみにこちら、変数にわざわざ格納しなくても、直接console出力は可能です！
```suggestion
console.log('私のニックネームは' + nickname + 'です。年齢は' + age + '歳です。');
```

Reply:
ありがとうございます！
わざわざ格納しなくても、コンソール出力可能とのこと承知いたしました...！